docs/troubleshooting/kubernetes.rst

```diff
@@ -25,3 +235,226 @@ If you've seen this error, you can override or change the default traffic-group
 .. code-block:: javascript
 
    "trafficGroup": "/Common/traffic-group-local-only"
+
+-----------------------------------------
+
+.. _logging troubleshoot:
+
+Logging
+```````
+
+Set the log level
+~~~~~~~~~~~~~~~~~
+
+To change the log level for the |kctlr|:
+
+#. Edit the :ref:`Deployment <k8s-bigip-ctlr-deployment>` for each k8s-bigip-ctlr instance. Be sure to specify the correct namespace for the instance.
+
+   .. code-block:: bash
+
+      kubectl edit my-bigip-ctlr.yaml [--namespace kube-system]
+      oc edit my-bigip-ctlr.yaml [--namespace kube-system]
+
+#. Add the desired log-level to the :code:`args` section, then save and exit the editor.
+
+   For example: ::
+
+   --log-level=debug
+
+#. Verify the Deployment updated successfully.
+
+   .. code-block:: bash
+
+      kubectl describe deployment my-bigip-ctlr -o wide [--namespace kube-system]
+      oc describe deployment my-bigip-ctlr -o wide [--namespace kube-system]
+
+-----------------------------------------
+
+View the logs
+~~~~~~~~~~~~~
+
+- To view the logs:
+
+  .. code-block:: bash
+
+     kubectl logs my-bigip-ctlr-pod
+     oc logs my-bigip-ctlr-pod
+
+- To follow the logs:
+
+  .. code-block:: bash
+
+     kubectl logs -f my-bigip-ctlr-pod
+     oc logs -f my-bigip-ctlr-pod
+
+- To view logs for a container that isn't responding:
+
+  .. code-block:: bash
+
+     kubectl logs --previous my-bigip-ctlr-pod
+     oc logs --previous my-bigip-ctlr-pod
+
+-----------------------------------------
+
+.. _networking troubleshoot:
+
+Networking
+``````````
+
+How do I verify connectivity between the BIG-IP VTEP Self IP and the OSE Node's VTEP IP?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Ping the Node's VTEP IP address.
+
+   Use the ``-s`` flag to set the MTU of the packets to allow for VxLAN encapsulation. ::
+
+     ping -s 1600 <OSE_Node_IP>
+
+#. Increase the log level to DEBUG and monitor the |kctlr| log files.
+
+   - Get the name of the k8s-bigip-ctlr for which you want to view logs. Be sure to specify the namespace the |kctlr| runs in. ::
+
+       oc get pod --namespace=kube-system
+       NAME                             READY     STATUS    RESTARTS   AGE
+       k8s-bigip-ctlr-687734628-7fdds   1/1       Running   0          15d
+
+   - Add the log-level=DEBUG annotation to the |kctlr| Deployment YAML file. ::
```

Comment:
I would re arrange this and move lines 321-324 to between lines 314-315.
because the oc get pod command is needed for viewing the logs not for setting the log level.